packages/shared/lib/core/network/utils/isIscpChain.ts

```diff
@@ -1,6 +1,6 @@
-import { ChainConfiguration } from '../types'
-import { IIscpChainConfiguration } from '../interfaces'
+import { IIscpChainMetadata } from '../interfaces'
+import { ChainMetadata } from '../types'
 
-export function isIscpChain(configuration: ChainConfiguration): configuration is IIscpChainConfiguration {
-    return (configuration as IIscpChainConfiguration) !== undefined
+export function isIscpChain(configuration: ChainMetadata): configuration is IIscpChainMetadata {
```

Comment:
Same as this one, we could probably simplify the logic around this as all chains will be iscp, and it will be very unlikely to have more than 1 isc chain configured, we could do this in another cleanup PR, wdyt?